### PR TITLE
Make paginate() compatible with core again

### PR DESCRIPTION
**Pull request by nckrtl**
This resolves https://github.com/statamic/cms/issues/6075#issuecomment-1134556716

### CODE REVIEW

**Comment by ryanmitchell** [comment]
Thanks, but this has already been fixed here: https://github.com/statamic/eloquent-driver/commit/8f00e25417da2d4766e22276bfac9027f0a74784

That PR is currently being reviewed and tested.

**Comment by nckrtl** [comment]
I've seen you mentioning that commit before, but as it shows the message that it's part of a different fork. I thought a PR has never been made. However, I see that the content of that commit became part of https://github.com/statamic/eloquent-driver/pull/42. 

You did an amazing job on that PR for sure, but I think commits like the one you mentioned could better be a single PR like this one. It helps to faster see what solutions have already been submitted (via the PR title/description). Especially because https://github.com/statamic/eloquent-driver/pull/42 is already so big and its hard to follow what's going on.

**Comment by nckrtl** [comment]
Hey @jasonvarga sorry to tag you as this might be on yours or your colleagues radar already. I just wanted to kindly give you a poke to see if this could get merged. If there is anything preventing this from getting merged I'm more than happy to look into it.

Projects that use this package are currently forced to use Statamic v3.3.11 instead of being able to use v3.3.18

**Comment by simonhamp** [comment]
I did this in #49 too and that was ignored. Now I'm bumping into this in production code and cannot wait for #42.

Guys, this is a simple fix to make this compatible again. Please can we just merge and release this so that everyone can get on with their day?

**Comment by simonhamp** [comment]
Thanks @jasonvarga 🙏🏼

**Comment by nckrtl** [comment]
Yeah thanks a lot!
